packages/rocketchat-lib/server/functions/getRoomByNameOrIdWithOptionToJoin.js

```diff
@@ -37,7 +38,11 @@ RocketChat.getRoomByNameOrIdWithOptionToJoin = function _getRoomByNameOrIdWithOp
 			}
 
 			room = Meteor.runAsUser(currentUserId, function() {
+<<<<<<< HEAD
```

Comment:
There's still a merge error here

Reply:
Cleared.  Feels Like I have now  done this exact same commit twice but maybe the first time is really still open  in a browser somewhere.

Reply:
Actualy, there were 2 like merge errors in this file, but the first time you only removed 1. This is all good now